### ISSUE TITLE
Don't download & import the example when running the server

### ISSUE THREAD

**Issue by maxboone** [vaexio/vaex#1690]
If you want to import the server to run it, it will always try to download the example dataframe. Even though this isn't explicitly needed.

Can we remove the line that executes this?
- https://github.com/vaexio/vaex/blob/0b5bc91fe45a8b39aaef170c8d9dcdc584715353/packages/vaex-server/vaex/server/fastapi.py#L389

**Comment by maartenbreddels** [comment]
Yeah, I think that should not be done by default, as we already have an options at https://github.com/vaexio/vaex/blob/0b5bc91fe45a8b39aaef170c8d9dcdc584715353/packages/vaex-server/vaex/server/fastapi.py#L439

What we could do, if 0 datasets exists, that it does do that.. so that you can always start the server, without reading docs etc. 

Do you want to open a PR?

**Comment by maxboone** [comment]
Sure thing, will do!
